confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -323,7 +323,7 @@ def get_compatibility(self, subject=None):
         if not is_successful_request:
             raise ClientError('Unable to fetch compatibility level. Error code: %d' % code)
 
-        compatibility = result.get('compatibility', None)
+        compatibility = result.get('compatibilityLevel', None)
```

Comment:
this seems to contradict the API docs:
https://docs.confluent.io/current/schema-registry/docs/api.html#put--config

code:
https://github.com/confluentinc/schema-registry/blob/8831bbd73ea720fcd279032a5fffabbdb1a1f2b1/client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/entities/Config.java#L25

Reply:
I have opened a docs jira for this. This absolutely the case I ran into the same issue implementing the SR client in golang. 

I had to handle it as follows: 

```
/* NOTE: GET uses compatibilityLevel, POST uses compatibility */
type compatibilityLevel struct {
	CompatibilityUpdate Compatibility `json:"compatibility,omitempty"`
	Compatibility       Compatibility `json:"compatibilityLevel,omitempty"`
}
```